.github/issue-regex-labeler.yml

```diff
@@ -0,0 +1,29 @@
+data:api:
```

Comment:
@queengooborg Note that these label names don't have the emojis, to reduce visual overload. We should remove the emojis from the existing labels before merging this.

Reply:
Added emojis in [514b998](https://github.com/mdn/browser-compat-data/pull/25059/commits/514b998b6d26d2b63bdd61b42e9cd24186860f55).